### PR TITLE
Allow MockConnector to create/rename/drop schema

### PR DESCRIPTION
**Pull request by Praveen2112**

### CODE REVIEW

**Review by skrzypo987 — APPROVED** [review]

File: core/trino-main/src/test/java/io/trino/connector/MockConnector.java [hunk 1]
```diff
@@ -261,6 +261,15 @@ public boolean schemaExists(ConnectorSession session, String schemaName)
...
         }
 
+        @Override
+        public void createSchema(ConnectorSession session, String schemaName, Map<String, Object> properties, TrinoPrincipal owner) {}
+
+        @Override
+        public void renameSchema(ConnectorSession session, String source, String target) {}
+
+        @Override
+        public void dropSchema(ConnectorSession session, String schemaName) {}
```
Comment (kokosing) [inline core/trino-main/src/test/java/io/trino/connector/MockConnector.java:271]: Are there any other like these? Alter views? Let's implement as much as possible so don't need to go back and forth.

**Review by kokosing — COMMENTED** [review]
Add some tests to make sure that actually mock connector supports that.

File: core/trino-main/src/test/java/io/trino/connector/MockConnector.java (hunk 1 above)
Comment (Praveen2112) [reply core/trino-main/src/test/java/io/trino/connector/MockConnector.java:271]: In case of `VIEW` I guess we can address them in a separate PR

**Comment by Praveen2112** [comment]
@kokosing AC

**Review by kokosing — APPROVED** [review]

**Pull request merged**